### ISSUE TITLE
doesnt recognize withSpeech module 

### ISSUE THREAD

**Issue by Jbry123** [ahrjarrett/react-speak#2]
```Cannot find module './src/withSpeech'```

**Comment by Jbry123** [comment]
I was reading a stackoverflow explaining how export default is for es6, and you said you were trying to do it for es5 correct? maybe if you just update to es6, or change the syntax, it would recognize it, and as always, check the file structure and how you're importing/exporting, im doing that right now.

**Comment by Jbry123** [comment]
```Jonathans-iMac:NodeTaskApp jonathanbry$ yarn add react-speak
yarn add v1.6.0
(node:98915) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
warning ../../../package.json: No license field
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Saved lockfile.
warning Your current version of Yarn is out of date. The latest version is "1.7.0", while you're on "1.6.0".
info To upgrade, run the following command:
$ brew upgrade yarn
success Saved 54 new dependencies.
info Direct dependencies
├─ bootstrap@3.3.7
├─ express@4.16.3
├─ mysql@2.15.0
└─ react-speak@2.0.1
info All dependencies
├─ accepts@1.3.5
├─ array-flatten@1.1.1
├─ asap@2.0.6
├─ bignumber.js@4.0.4
├─ body-parser@1.18.2
├─ bootstrap@3.3.7
├─ content-disposition@0.5.2
├─ cookie-signature@1.0.6
├─ cookie@0.3.1
├─ core-js@1.2.7
├─ core-util-is@1.0.2
├─ destroy@1.0.4
├─ ee-first@1.1.1
├─ encoding@0.1.12
├─ express@4.16.3
├─ finalhandler@1.1.1
├─ forwarded@0.1.2
├─ inherits@2.0.3
├─ ipaddr.js@1.6.0
├─ is-stream@1.1.0
├─ isarray@1.0.0
├─ isomorphic-fetch@2.2.1
├─ js-tokens@3.0.2
├─ loose-envify@1.3.1
├─ media-typer@0.3.0
├─ merge-descriptors@1.0.1
├─ methods@1.1.2
├─ mime-db@1.33.0
├─ mime@1.4.1
├─ mysql@2.15.0
├─ negotiator@0.6.1
├─ node-fetch@1.7.3
├─ object-assign@4.1.1
├─ path-to-regexp@0.1.7
├─ process-nextick-args@1.0.7
├─ promise@7.3.1
├─ prop-types@15.6.1
├─ proxy-addr@2.0.3
├─ raw-body@2.3.2
├─ react-speak@2.0.1
├─ react@16.4.0
├─ readable-stream@2.3.3
├─ safer-buffer@2.1.2
├─ serve-static@1.13.2
├─ setimmediate@1.0.5
├─ sqlstring@2.3.0
├─ string_decoder@1.0.3
├─ type-is@1.6.16
├─ ua-parser-js@0.7.18
├─ unpipe@1.0.0
├─ util-deprecate@1.0.2
├─ utils-merge@1.0.1
├─ vary@1.1.2
└─ whatwg-fetch@2.0.4
✨  Done in 10.01s.
Jonathans-iMac:NodeTaskApp jonathanbry$ 
```

**Comment by Jbry123** [comment]
this is what i got when i downloaded with yarn, is it just when you use it?

**Comment by ahrjarrett** [comment]
Thank you Jon! Fixed in version 2.0.0
